doc/advanced-testing-mini-course/app/Main.hs

```diff
@@ -51,7 +52,8 @@ main = do
           printItem "Sent messages" "" ""
           mapM_ (\x -> putStrLn $ "  " <> show x) msgs
         fs = FailureSpec (NetworkFaults 0.15)
-      collector <- eventLoopFaultySimulation (Seed 6) VR.agenda h fs
+        endTime = addTimeMicros (600*10^6) epoch
```

Comment:
```suggestion
        endTime = addTime 600 epoch
```

Reply:
We should probably rename `addTime` to `addTimeSeconds`.

Reply:
Ah right. Thanks